### PR TITLE
Add cookies and privacy policy pages

### PR DESCRIPTION
**Pull request by dashouse**
- Add Cookies page
- Add Privacy policy page
- Add links to both in the footer
- Add Cookie banner to the DS (SCSS, JS and Generic layout)
- Amend single page layout to use large wrapper variant

### CODE REVIEW

**Comment by dashouse** [comment]
This needs a technical review from a dev and checked over by @alicenoakes

**Comment by alicenoakes** [comment]
Looks good

File: src/cookies.md.njk [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+  "rows": [
+    [
+      {
+        "text": "seen_cookie_message"
+      },
+      {
+        "text": "Saves a message to let us know that you have seen our cookie message"
+      },
+      {
+        "text": "1 month"
```
Comment (igloosi) [inline src/cookies.md.njk]: code says 28 days

**Comment by dashouse** [comment]
Merged, thanks

**Pull request merged**